aict_tools/plotting.py

```diff
@@ -130,24 +133,31 @@ def plot_roc(performace_df, model, ax=None):
     return ax
 
 
-def plot_probabilities(performace_df, model, ax=None, classnames=('Proton', 'Gamma')):
+def plot_probabilities(performace_df, model, ax=None, classnames={0:'Proton', 1:'Gamma'}, label_str='label', label_proba_str='probabilities'):
 
     ax = ax or plt.gca()
 
     if isinstance(model, CalibratedClassifierCV):
         model = model.base_estimator
 
     n_bins = (model.n_estimators + 1) if hasattr(model, 'n_estimators') else 100
-    bin_edges = np.linspace(0, 1, n_bins + 1)
+    if label_proba_str == 'sign_score':
+        bin_edges = np.linspace(-1, 1, n_bins + 1)
+    else:
+        bin_edges = np.linspace(0, 1, n_bins + 1)
 
-    for label, df in performace_df.groupby('label'):
+    for label, df in performace_df.groupby(label_str):
         ax.hist(
-            df.probabilities,
+            df[label_proba_str],
             bins=bin_edges, label=classnames[label], histtype='step',
         )
 
-    ax.legend()
-    ax.set_xlabel('{} confidence'.format(classnames[1]))
+    if label_proba_str == 'sign_score':
+        ax.set_xlabel('Score')
```

Comment:
Score is fine for everything. You could give the xlabel as argument and take it from the `target_name` config option

Reply:
I put `score` as default and in `plot_separator_performance.py` the config option is given.